src/components/Tooltip/index.tsx

```diff
@@ -87,9 +88,9 @@ export function MouseoverTooltip({ children, disableTooltip, delay, ...rest }: O
   if (disableTooltip) return <>{children}</>
   return (
     <Tooltip {...rest} show={show} onMouseEnter={open} onMouseLeave={close}>
-      <Flex onMouseOver={open} onMouseLeave={close} alignItems="center">
+      <Row onMouseOver={open} onMouseLeave={close}>
```

Comment:
double check it. Row always has width 100%, it may break some UI

Reply:
already restricted by TooltipContainer's width